src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = Pick<Movie, 'title' | 'description' | 'imgUrl' | 'imdbUrl'>;
```

Comment:
It's redundant here, you can get all Movie, and in destructuring get needed props
```suggestion
type Props = {
  movie: Movie;
};
```